packages/frontend/src/contexts/AppContext/AppContext.tsx

```diff
@@ -22,6 +22,8 @@ type AppContextProps = {
   setSelectedBridge: (bridge: HopBridge) => void
   user: User | undefined
   networks: Network[]
+  l2Networks: Network[]
+  DEFAULT_L2_NETWORK: Network | undefined
```

Comment:
I think this should be treated as a regular read only property (`defaultL2Network`) instead of an uppercased constant since this value is computed and is not a hard coded constant